docs/customizing.rst

```diff
@@ -117,3 +117,33 @@ relevant rules:
    pre, code, kbd, samp {
      font-family: "Courier New", monospace;
    }
+
+Custom Sidebars
+---------------
+
+To add a custom sidebar (for example, add a link to sponsor your project),
+create a custom HTML template and specify it in ``conf.py``:
```

Comment:
I see that you're following Sphinx's nomenclature, but I wonder if we could elaborate a little here to make it clear that in the context of this theme, what these instructions actually do is add a new _section_ to the main/left sidebar.

The explanation of "sidebars" in the Sphinx docs is puzzling to me, at least at this late hour, so I may be misunderstanding something about what exactly they are to Sphinx, but my initial assumption about what "adding a custom sidebar" would mean is replacing the sidebar entirely, so that's why I was a little thrown off by this.

Reply:
Good point on wording of it.  You are correct: the sidebar is one entity, each section within the sidebar is its own separate template. So you can customize the sidebar by rearranging, adding, removing, etc. templates.

I will update the doc for clarity.